extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakBuildTimeConfig.java

```diff
@@ -12,5 +17,32 @@ public class KeycloakBuildTimeConfig {
      * Dev services configuration.
      */
     @ConfigItem
-    public DevServicesConfig devservices;
+    public KeycloakDevServicesConfig devservices;
```

Comment:
I'm a bit confused what is happening here. Well, I think you have some interesting ideas here, but updating OIDC Dev Services should be, IMHO, a separate issue with a dedicated discussion. IMHO this PR will become much simpler :-) if we focus here on supporting a basic `Keycloak` injection, and then we can continue from there

Reply:
Well, this simply adds configuration option for client roles and default client scope as explained in the docs. How could you manage Keycloak with `microprofile-jwt` and without possibility to add users created by dev services new client roles (view users, view roles).

I can remove it, but once again, whole point of Dev Services is to allow user hit the ground running. With single line (as clearly showed in UsersConfiguratios test I added) allows user to configure. Doing same with keycloak.json takes much longer.

Reply:
And it's more complicated, not every developer is Keycloak expert...

Reply:
@michalvavrik Sure, I'd just prefer this enhancement it done in a follow up PR as this one is only about  supporting an injection as opposed to a manual construction of Keycloak